runtime/parser2/lexer/state.go

```diff
@@ -210,7 +210,7 @@ func numberState(l *lexer) stateFn {
 			}
 			l.emitValue(TokenHexadecimalIntegerLiteral)
 
-		case '0', '1', '2', '3', '4', '5', '6', '7', '8', '9':
+		case '0', '1', '2', '3', '4', '5', '6', '7', '8', '9', '_':
```

Comment:
would this allow numbers starting with `_`? e.g: `_0`?

Reply:
No, this case only may occur after an initial `0`, see https://github.com/onflow/cadence/pull/1348/files#diff-e258ba2fee01ca54837c0062a323e801197f430aeff7ce6185b0130f51b511beR189 above

Reply:
Added in 71d7af0d0707dd41c745b2ba8c91c410b231626d

Reply:
Thank you!